app/controllers/UnauthorisedController.scala

```diff
@@ -32,3 +34,20 @@ class UnauthorisedController @Inject()(
     Ok(view())
   }
 }
+
+object UnauthorisedController {
+
+  def viewModel(): FormPageViewModel[ContentPageViewModel] =
+    FormPageViewModel(
+      Message("unauthorised.title"),
+      Message("unauthorised.heading"),
+      ContentPageViewModel(isLargeHeading = false),
+      routes.UnauthorisedController.onPageLoad()
+    ).withDescription(
+      Message("unauthorised.paragraph") ++
+        LinkMessage(
+          Message("unauthorised.linkMessage"),
+          "https://www.gov.uk/government/organisations/hm-revenue-customs/contact/pension-scheme-enquiries"
```

Comment:
We already have this in a configuration and have a property for it: `pensionSchemeEnquiry`